WcaOnRails/app/models/competition.rb

```diff
@@ -809,6 +805,25 @@ def self.search(query, params: {}, managed_by_user: nil)
     competitions.includes(:delegates, :organizers).order(start_date: :desc)
   end
 
+  # See https://github.com/thewca/worldcubeassociation.org/wiki/wcif
+  def to_wcif
+    managers = (organizers + delegates).uniq
```

Comment:
Why not use the existing `def managers` for this? See https://github.com/thewca/worldcubeassociation.org/blob/e0b44127bcd7fbcf547393a63edac2a83cc342aa/WcaOnRails/app/models/competition.rb#L413-L415